src/components/component-content/ComponentContent.tsx

```diff
@@ -3,16 +3,22 @@ import TOC from "@components/table-of-contents/TOC";
 
 type Props = {
   children: React.ReactNode;
-  cssQuery: string;
+  tocCssQuery?: string;
```

Comment:
What is the reason for changing this prop name?

Reply:
After seeing the changes to the ComponentContent I can see why the change was done.

Reply:
Thank you @chrisolsen